spec/factories/dishes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :dish do
+    sequence(:creator_id) { |n| }
```

Comment:
You have to be signed in to create a dish, and a creator is just an alias for a user! Hence, creator is the current user when a dish is created